templates/test/live.html.ep

```diff
@@ -7,7 +7,7 @@
     <tr>
         <td>Interactive:</td>
         <td id="interactive_indicator" data-next-status="">Unknown</td>
-        <td id="needinput_buttons">
+        <td>
```

Comment:
Because an ID must be unique within an HTML document and those IDs aren't used anyways.